dask/dataframe/io/csv.py

```diff
@@ -356,6 +356,10 @@ def read_pandas(reader, urlpath, blocksize=AUTO_BLOCKSIZE, collection=True,
 
     if not isinstance(values[0], (tuple, list)):
         values = [values]
+    # If we have not sampled, then use the first row of the first values
+    # as a representative sample.
+    if b_sample is False and len(values[0]):
+        b_sample = values[0][0].compute()
```

Comment:
I like this as a very simple way to achieve things. 

The docstring here needs to be updated.

Reply:
Do we still get an appropriate exception when `values[0] == []`?

Reply:
Good question!

Reply:
I'm not really sure what the intended behavior should be here. When the CSV is empty it raises a `ValueError` that the sample is not large enough.

Reply:
I guess it's not the time to change the behaviour here, so it's fine - but this is about inferring the meta from the first file, but the first file might not contain sufficient or any data; in theory we could deal with that, but probably the error message is good enough.